background/background.go

```diff
@@ -209,6 +211,14 @@ func blockConnected() {
 					if err != nil {
```

Comment:
Because we do expect this scenario to happen occasionally, and we have appropriate handling for it, I think we should avoid logging errors and just log some warnings instead.

```
err = setVoteChoice
if error {
    if "no agenda with ID" {
        log.warn "Removing invalid agenda from ticket vote choices (ticketHash=blah, agenda=blah)"
        delete choice
    } else {  // any other error
       log.error
    }
}
```